src/nix.rs

```diff
@@ -0,0 +1,49 @@
+use libc::c_char;
```

Comment:
As we're already exporting the contents of this module on the top level, I've made this module private..